src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -340,7 +333,7 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
     // The following if statements are necessary to support the context bar properly loading runtimes for google/azure
     if (workspaceLoaded) {
       if (googleProject !== prevGoogleProject || azureContext !== prevAzureContext) {
-        refreshRuntimes(isAzureWorkspace(workspace))
+        refreshRuntimes(true)
```

Comment:
Cleanup discussed in an older PR-- https://github.com/DataBiosphere/terra-ui/pull/3702/files/65b6a66ad1f996447439620e45ef6c20aaf455b9#r1082910869